MVP_Clicker/Assets/Project/Scripts/Core/LoadResourcesService/AddressableLoadResourceService.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using UnityEngine.AddressableAssets;
+
+namespace Project.Scripts.Core.LoadResourcesService
+{
+    public class AddressableLoadResourceService : ILoadResourcesService
+    {
+        private readonly Dictionary<string, object> _resources = new();
+        private readonly Dictionary<string, int> _callsCounter = new();
+
+        public T Load<T>(string path)
```

Comment:
Your method is not async. When some resouce will be load it will stop the game untill it will finish game. Are you sure you want such behaviour?